__init__.py

```diff
@@ -288,23 +327,19 @@ def __init__(self):
         self.CODES['50d', '50n'] = 7                # windy/misty
 
         # Use Mycroft proxy if no private key provided
-        self.settings["api_key"] = None
-        self.settings["use_proxy"] = True
+        self.settings.setdefault("api_key", None)
+        self.settings.setdefault("use_proxy", True)
 
     def initialize(self):
-        # TODO: Remove lat,lon parameters from the OWMApi()
-        #       methods and implement _at_coords() versions
-        #       instead to make the interfaces compatible
-        #       again.
-        #
-        # if self.settings["api_key"] and not self.settings['use_proxy']):
-        #     self.owm = OWM(self.settings["api_key"])
-        # else:
-        #     self.owm = OWMApi()
-        self.owm = OWMApi()
+        # lat,lon parameters are handled by OWMApi() both for
+        # local api key and mycrodt's api
```

Comment:
If we agree on this approach, then I don't know that we need this comment here any more.

Reply:
and would also kill the spelling mistake